shared/src/persistence/elasticsearch/getIndexedCasesForUser.js

```diff
@@ -32,19 +32,13 @@ exports.getIndexedCasesForUser = async ({
         bool: {
           must: [
             {
-              match: {
-                'pk.S': { operator: 'and', query: `user|${userId}` },
+              term: {
+                'pk.S': `user|${userId}`,
```

Comment:
Perhaps this should be done in a follow up ticket, but i would love to see the PK and SKs dropped in favor of the UUIDs being their own prop. I realize this would affect the processStreamsInteractor since that what does the indexing.